ArcGISRuntimeSDKQt_CppSamples/Search/FindPlace/FindPlace.h

```diff
@@ -29,6 +29,7 @@ namespace Esri::ArcGISRuntime
 #include "Point.h"
 #include "Envelope.h"
 #include "GeocodeParameters.h"
+#include "GeocodeResult.h"
```

Comment:
This should be forward declared. Everything should be to the fullest extent possible.

Reply:
Done.